packages/react/src/components/icon/icon.stories.tsx

```diff
@@ -15,20 +22,29 @@ export default {
         type: { summary: colors.map((value) => `"${value}"`).join('|') },
       },
     },
+    name: {},
+    'aria-hidden': aria.hidden,
+    'aria-label': aria.label,
   },
   args: {
     name: firstIconName,
   },
 } as Meta<IconProps>;
 
 export const Playground: Story<IconProps> = (props) => <Icon {...props} />;
+Playground.args = {
+  'aria-label': 'a label for my icon',
```

Comment:
Following previous accepted suggestion:

```suggestion
  'aria-label': 'A label for an icon',
```